store/db_instance.go

```diff
@@ -83,6 +83,20 @@ func Convert(offerList []*client.Offer, cloudProvider CloudProvider) ([]*DBInsta
 			CommitmentUSD:  offer.CommitmentUSD,
 		}
 		termMap[offer.ID] = append(termMap[offer.ID], term)
+		// This is a little bit hack.
+		// GCP charge MySQL and PostgreSQL equally, but only provide MySQL at their API message.
+		// We manually create a PostgreSQL with exactly the same price here.
+		if cloudProvider == "GCP" {
```

Comment:
This looks strange since store/db_instance shouldn't be tied to cloud provider specifics. This should belong to GCP client if possible.